Core/NoOpRunner.Core/PlayerStates/PlayerOneStateMachine.cs

```diff
@@ -1,29 +1,34 @@
 ﻿using System;
-using NoOpRunner.Core.Enums;
 
 namespace NoOpRunner.Core.PlayerStates
 {
     public class PlayerOneStateMachine
     {
-        private PlayerOneState currentState = PlayerOneState.Idle;
-        
-        public PlayerOneState State
+        private PlayerState CurrentState = new IdleState();
```

Comment:
**[Nitpick]** Since `CurrentState` is a field, not a property, the name should be camelCase